shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -32,6 +32,16 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
 }
 @end
 
+static bool ClipBoundsContainsPlatformViewBoundingRect(const SkRect& clipBounds,
+                                                       const SkRect& platformViewBoundingRect,
```

Comment:
Optional nit for here and the local variables: while either snake_case or camelCase are allowed by style for C(++) functions in ObjC++ code, it looks like most of the surrounding code here is C++ using snake_case, so it seems like snake_case for these would be more consistent.